### PR TITLE
EZSP v10

### PR DESCRIPTION
**Pull request by puddly**
The only major addition is `incomingNetworkStatusHandler`, which was previously `changeSourceRouteHandler`. I haven't seen any use for it:

```Python
DEBUG (MainThread) [bellows.ezsp.protocol] Application frame received incomingNetworkStatusHandler: [<EmberStackError.ROUTE_ERROR_MANY_TO_ONE_ROUTE_FAILURE: 12>, 0xafa8]
DEBUG (MainThread) [bellows.ezsp.protocol] Application frame received incomingNetworkStatusHandler: [<EmberStackError.ROUTE_ERROR_INDIRECT_TRANSACTION_EXPIRY: 6>, 0x9253]
```

### CODE REVIEW

File: bellows/ezsp/v10/__init__.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+from . import commands, config, types as v10_types
+from .. import protocol
+
+EZSP_VERSION = 10
+LOGGER = logging.getLogger(__name__)
+
+
+class EZSPv10(protocol.ProtocolHandler):
+    """EZSP Version 8 Protocol version handler."""
```
Comment (TheJulianJES) [inline bellows/ezsp/v10/__init__.py:18]:
```suggestion
class EZSPv10(protocol.ProtocolHandler):
    """EZSP Version 10 Protocol version handler."""
```

**Comment by MattWestb** [comment]
In the current `UG100: EZSP Reference Guide` its the only change but its still V9 and is not updated.
I can being that V9 is not making the change and some is getting problems that the old command is used and not the new one and Silabs was bumping the protocol for blocking use of the old command that is making problems in some system (like Z2M).

Shall i posting one question in Silabs community and asking for information ??
If its not working its also possible putting one support ticket but i think it shall not being needed.

**Comment by Hedda** [comment]
> The only major addition is `incomingNetworkStatusHandler`, which was previously `changeSourceRouteHandler`. I haven't seen any use for it

https://github.com/home-assistant/core/issues/85822 does mention those?

**Comment by Hedda** [comment]
For reference also see https://github.com/Koenkk/zigbee-herdsman/pull/640 and https://github.com/Koenkk/zigbee2mqtt/issues/15724 and https://github.com/home-assistant/core/issues/85822

**Comment by Hedda** [comment]
FYI, updated PR https://github.com/zigpy/bellows/pull/463 which adds a mentioning of EZSP v9 and EZSP v10 in the README.md for bellows.

**Comment by MattWestb** [comment]
Time for one more update ;-)
`Protocol version 11 is not supported, using version 10 instead`
Its little to much coding for my so i letting it for some one that can doing it right from the geginning.

PS: Looks like the updated addon is using the save version of CPC V3 so GSDK 4.2.1.0 is still working and shall doing with dongle with 4.2.2.0 on addon 1.0.1

**Pull request merged**